sys/net/gnrc/link_layer/lorawan/gnrc_lorawan_mlme.c

```diff
@@ -27,11 +27,25 @@
 
 #include "net/lorawan/hdr.h"
 
-#define ENABLE_DEBUG 0
+#define ENABLE_DEBUG 1
 #include "debug.h"
 
-static void _build_join_req_pkt(uint8_t *appeui, uint8_t *deveui,
-                                uint8_t *appkey, uint8_t *dev_nonce,
+#if IS_ACTIVE(CONFIG_LORAWAN_1_1)
+/**
+ * @brief 2-byte counter
+ *
+ * Starting at 0 when device is powered up and incremented with every
+ * Join-request.
+ *
+ * TODO: A DevNonce value SHALL NEVER be reused for a given JoinEUI value.
+ * If the end-device can be power-cycled then DevNonce SHALL be persistent
+ * (stored in a non-volatile memory)
+ */
+static uint16_t _dev_nonce = 0x0;
```

Comment:
Is there a reason to put this here rather than in the MAC descriptor? (`gnrc_lorawan_t`).

Reply:
No actually not. Removed it